src/test/lambda_log_shipper/test_configuration.py

```diff
@@ -28,3 +28,23 @@ def test_parse_env_to_int_exists(actual_env, expected_result, monkeypatch):
 def test_parse_env_to_int_not_exists(monkeypatch):
     monkeypatch.delenv("ENV_VAR", None)
     assert parse_env_to_int("ENV_VAR", 5) == 5
+
+
+@pytest.mark.parametrize(
+    "actual_env, expected_result",
+    [
+        ("False", False),
+        ("True", True),
+        ("false", False),
+        ("true", False),
```

Comment:
This test fails

Reply:
Sorry about that. Looks like I pushed the fix up to the wrong branch before. This should now be ok.